### ISSUE TITLE
consider a check to fail use of aria-selected if not inside an allowed container role

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#102]
containers for aria-selected should have 
role="tab"
role=gridcell
role=option
https://www.w3.org/TR/wai-aria-1.1/#aria-selected
or inherited into some grid roles

otherwise should use aria-current
https://www.w3.org/TR/wai-aria-1.1/#aria-current

**Comment by irwink** [comment]
Feature added in version 6.9.0.
